pulpcore/cli/python/remote.py

```diff
@@ -0,0 +1,54 @@
+import gettext
+
+import click
+
+from pulpcore.cli.common.context import PulpContext, pass_pulp_context
+from pulpcore.cli.common.generic import (
+    create_command,
+    destroy_command,
+    href_option,
+    label_command,
+    label_select_option,
+    list_command,
+    name_option,
+    show_command,
+    update_command,
+)
+from pulpcore.cli.python.context import PulpPythonRemoteContext
+
+_ = gettext.gettext
+
+
+@click.group()
+@click.option(
+    "-t",
+    "--type",
+    "remote_type",
+    type=click.Choice(["python"], case_sensitive=False),
+    default="python",
+)
+@pass_pulp_context
+@click.pass_context
+def remote(ctx: click.Context, pulp_ctx: PulpContext, remote_type: str) -> None:
+    if remote_type == "python":
+        ctx.obj = PulpPythonRemoteContext(pulp_ctx)
+    else:
+        raise NotImplementedError()
+
+
+lookup_options = [href_option, name_option]
+create_options = [
+    click.option("--name", required=True),
+]
+python_remote_options = [
+    click.option("--url"),
+    click.option("--includes", help=_("Package allowlist")),
+    click.option("--excludes", help=_("Package blocklist")),
```

Comment:
Are those also JSON lists?

Reply:
Yes they are json lists

Reply:
So they can use the json option callback once we merged the access policy PR.
#104